src/test/TestUtils/DotNetCli.cs

```diff
@@ -67,7 +67,8 @@ public Command Exec(string command, params string[] args)
             }
 
             return Command.Create(DotnetExecutablePath, newArgs)
-                .EnvironmentVariable("DOTNET_SKIP_FIRST_TIME_EXPERIENCE", "1");
+                .EnvironmentVariable("DOTNET_SKIP_FIRST_TIME_EXPERIENCE", "1")
+                .EnvironmentVariable("DOTNET_MULTILEVEL_LOOKUP", "0"); // Avoid looking at machine state by default
```

Comment:
Nit: Why not `.MultiLevelLookup(false)` ?

Reply:
This is in TestUtils and our extensions are in HostActivation.Tests. (Admittedly, I didn't really look into moving things about in this change)

Reply:
Right - sorry.